easybuild/easyblocks/c/clang.py

```diff
@@ -98,6 +100,7 @@ def extra_options():
             # The sanitizer tests often fail on HPC systems due to the 'weird' environment.
             'skip_sanitizer_tests': [True, "Do not run the sanitizer tests", CUSTOM],
             'usepolly': [False, "Build Clang with polly", CUSTOM],
+            'llvm_projects': [[], "LLVM projects to install", CUSTOM],
```

Comment:
Should we curate this list and have accepted values? With this PR we would support
```
llvm_projects = ['polly', 'lld', 'libunwind', 'lldb', 'libcxx', 'libcxxabi', 'clang-tools-extra']
```
but what if someone goes outside this group? We could let them do it but throw a warning that the project is not currently supported by the easyblock but that we will include it in the build anyway.